src/test/java/bio/terra/app/usermetrics/UserMetricsInterceptorTest.java

```diff
@@ -167,6 +167,33 @@ void testSendEventWithAdditionalProperties() throws Exception {
     assertThat("token is correct", authCaptor.getValue().getToken(), equalTo(TOKEN));
   }
 
+  @Test
```

Comment:
Thanks for adding a unit test!

Could you please also add a test for the condition when the property is not present in the header, or is that already covered by an existing unit test?

Reply:
I think the test would look exactly like`testSendEvent`, which does not have any headers set.